tst/com/amazon/corretto/crypto/provider/test/AesCbcTest.java

```diff
@@ -497,7 +485,7 @@ public void testMultiStepByteBuffer(
   private static Stream<Arguments> byteBufferTestParams() {
     final List<Arguments> result = new ArrayList<>();
     for (final int keySize : new int[] {128}) {
-      for (int i = 0; i != 1024; i++) {
```

Comment:
In the Git commit I mentioned this. Some of the MacOS CI tasks sometimes run out of memory with this test.

Reply:
This is from the CBC tests and it's not part of this change. The ISO10126 implementation doesn't allocate anything on the heap and it relies on CBC implementation. If there is any leaks, it must be coming from the CBC implementation. I can see one potential leak from in the CBC implementation in the following two places: [InitUpdate](https://github.com/corretto/amazon-corretto-crypto-provider/blob/main/src/com/amazon/corretto/crypto/provider/AesCbcSpi.java#L366), and [InitUpdateFinal](https://github.com/corretto/amazon-corretto-crypto-provider/blob/main/src/com/amazon/corretto/crypto/provider/AesCbcSpi.java#L538)
`new NativeEvpCipherCtx` registers the allocated `EVP_CIPHER_CTX` with the Janitor. In case the registration fails, we could have a leak. However, this style of allocate- and-then-register has been the norm in other parts of our code: https://github.com/corretto/amazon-corretto-crypto-provider/blob/main/src/com/amazon/corretto/crypto/provider/AesGcmSpi.java#L1004